src/index.js

```diff
@@ -0,0 +1,169 @@
+// const newProjectForm = document.querySelector('.new_project');
+const newProjectInput = document.querySelector('[data-new-project-input]');
+// const newTaskInput1 = document.querySelector('[data-new-todo-form]');
+// const newTaskInput = document.getElementsByClassName('form-group');
+
+const newTodoInputTitle = document.querySelector('[data-new-todo-title-input]');
+const newTodoInputDesc = document.querySelector('[data-new-todo-desc-input]');
+const newTodoInputPrior = document.querySelector('[data-new-todo-prior-input]');
+const newTodoInputDate = document.querySelector('[data-new-todo-date-input]');
+const newTodoInputTime = document.querySelector('[data-new-todo-time-input]');
+const newTodoInputNote = document.querySelector('[data-new-todo-note-input]');
+
+const addProject = document.getElementById('add_project');
+const addTask = document.getElementById('add_task');
+
+const projectForm = document.getElementById('project_form');
+const taskForm = document.getElementById('task_form');
+
+const deleteTask = document.getElementById('button_delete');
+const updateTask = document.getElementById('button_update');
+
+const LOCAL_STORAGE_ALL_PROJECTS = 'projects';
+
+const projects = JSON.parse(localStorage.getItem(LOCAL_STORAGE_ALL_PROJECTS)) || [
+  {
+    id: Date.now().toString(),
+    name: 'Default Project',
+    todos: [],
+  },
+];
+
+const renderdescription = (selectedTaskId1, selectedProject) => {
+  descriptionContainer.innerHTML = '';
+  const selectedTask = selectedProject.todos.find((task) => task.id === selectedTaskId1);
+  descriptionContainer.innerHTML = `Name:- ${selectedTask.name} <br> Description:-${selectedTask.desc}
+        <br> Priority:- ${selectedTask.prior} <br> Date:- ${selectedTask.date} <br> note:- ${selectedTask.note} <br>
+         `;
+  deleteTask.style.visibility = 'visible';
+  updateTask.style.visibility = 'visible';
+};
+
+function description(task, selectedProject) {
+  const selectedTaskId1 = task.id;
+  renderdescription(selectedTaskId1, selectedProject);
+  return selectedTaskId1;
+}
+
+const rendertask = (selectedProjectId1) => {
+  taskContainer.innerHTML = '';
+  descriptionContainer.innerHTML = '';
+  deleteTask.style.visibility = 'hidden';
+  updateTask.style.visibility = 'hidden';
+  const selectedProject = projects.find((project) => project.id === selectedProjectId1);
+  taskContainer.innerHTML = `Project Name:-${selectedProject.name}`;
+  selectedProject.todos.forEach((task) => {
+    const taskElement = document.createElement('li');
+    taskElement.dataset.taskId = task.id;
+    taskElement.innerText = task.name;
+    taskElement.onclick = () => { selectedTaskId1 = description(task, selectedProject); };
+    taskContainer.appendChild(taskElement);
+  });
+};
+
+function task(project) {
+  addTask.style.visibility = 'visible';
+  const selectedProjectId1 = project.id;
+  rendertask(selectedProjectId1);
+  return selectedProjectId1;
+}
```

Comment:
- [ ] Kindly separate this logic from the DOM by moving by placing it in a different file. Use import and export statements to use the logic in `index.js`
- [ ] Kindly use the ES6 arrow function instead of the function keyword